plugins/home/src/components/HomeIndexPage.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { ReactNode } from 'react';
+import { useOutlet } from 'react-router';
+
+export const HomeIndexPage = (props: { children?: ReactNode }) => {
```

Comment:
What's the point of this component? I thought it might be a placeholder for providing more functionality like drag and drop or dynamic addition of home page widgets, but in that case I think you'd want to wrest a bit more control and have the `<Page>`, `<Content>`, etc. type wrappers in here instead.

I can imagine something that requires a bit less layout on the part of integrators, like:

```tsx
<Route path="/" component={<HomePage />}>
  <RandomJokeCard />
  <GithubPullRequestsCard />
  <GithubLatestActionsCard count={10} />
</Route>
```

This would assume a fixed card layout initially, though I could see passing props to each card as an integrator or eventually having users in control of the layout here.